### DISCUSSION TITLE
Processing your files… This may take a few minutes to complete. - takes forever with upload of external files

### DISCUSSION THREAD

**Discussion by pibinko** [community/community#44497] [New to GitHub]
Hello,

I have started a simple repository a few days ago.

I have now tried to upload a new external file (2Kb in size), and when requesting the "commit" I see the usual message "Processing your files… This may take a few minutes to complete".  but then nothing happens for minutes.

If I create a new file directly in the repository, the commit takes a few seconds.

I did some research on the topic and see the is mentioned in past bugs for github developers, but I could not find a solution to be applied as a basic user. TIA for any feedback

**Comment by michaelhasey** [comment]
Same Issue.  Completely halted my workflow.  Hoping this can be fixed asap!

**Comment by pibinko** [comment]
Thanks Michael ...so this is not just related to my user..I also noted that there is a message "GitHub Service Status
Actions is currently status yellow; with an update as of about 1 hour ago. This may affect Actions behavior and performance.
Please check [GitHub Status](https://stspg.io/sx6vljvfq39c) for more information." ....so a resolution should be on its way.

Best regards!

**Comment by michaelhasey** [comment]
Ah, I missed that message.  Thanks for the update and looking forward to a solution!

**Comment by queenofcorgis** [comment]
Hi all - are you still experiencing this?

**Reply by pibinko** [reply]
Hello - this seems ok now. Best regards! BTW is there a way to close the thread?
